kolibri_explore_plugin/views.py

```diff
@@ -109,6 +113,35 @@ def get(self, request, app):
             return HttpResponse(json_file, content_type="application/json")
 
 
+def _import_channel_from_usb(channel_id, directory):
```

Comment:
Function name nitpick: this could be any fallback directory so I would change "usb" here.
```suggestion
def _import_channel_from_directory(channel_id, directory):
```